src/gui/mzroll/ligandwidget.cpp

```diff
@@ -61,17 +62,17 @@ LigandWidget::LigandWidget(MainWindow* mw) {
   connect(this, SIGNAL(compoundFocused(Compound*)), mw, SLOT(setCompoundFocus(Compound*)));
   connect(this, SIGNAL(urlChanged(QString)), mw, SLOT(setUrl(QString)));
 
-  saveButton = new QToolButton(toolBar);
-  saveButton->setIcon(QIcon(rsrcPath + "/filesave.png"));
-  saveButton->setToolTip("Save Compound List");
-  connect(saveButton,SIGNAL(clicked()), SLOT(saveCompoundList()));
-
-
-  saveButton->setEnabled(false);
+  libraryButton = new QToolButton(toolBar);
+  libraryButton->setIcon(QIcon(rsrcPath + "/librarymanager.png"));
+  libraryButton->setToolTip("Open Library Manager");
+  connect(libraryButton,
+          &QPushButton::clicked,
+          _mw->getLibraryManager(),
+          &LibraryManager::exec);
 
   toolBar->addWidget(databaseSelect);
   toolBar->addWidget(loadButton);
```

Comment:
@saifulbkhan We can remove the load button now that the library widget exists. It's in the same position as the previous button so shouldn't be hard to find.